rpm/pacemaker.spec.in

```diff
@@ -174,19 +164,13 @@
                             sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g'; })
 %endif
 
-## Values that differ by Python major version
-%if 0%{?python_major} > 2
+## Prefer Python 3 definitions explicitly, in case 2 is also available
+%if %{defined __python3}
 %global python_name python3
-%global python_path %{?__python3}%{!?__python3:/usr/bin/python%{?python3_pkgversion}%{!?python3_pkgversion:3}}
+%global python_path %{__python3}
 %define python_site %{?python3_sitelib}%{!?python3_sitelib:%(
   %{python_path} -c 'from distutils.sysconfig import get_python_lib as gpl; print(gpl(1))' 2>/dev/null)}
 %else
-%if 0%{?python_major} > 1
-%global python_name python2
-%global python_path %{?__python2}%{!?__python2:/usr/bin/python%{?python2_pkgversion}%{!?python2_pkgversion:2}}
-%define python_site %{?python2_sitelib}%{!?python2_sitelib:%(
-  %{python_path} -c 'from distutils.sysconfig import get_python_lib as gpl; print(gpl(1))' 2>/dev/null)}
-%else
 %if %{defined python_version}
```

Comment:
We'll need to be on the lookout to make sure these changes don't break anything on SuSE.  This was a whole complicated area.

Reply:
good news is opensuse-15 is building